package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "node-tonic",
+  "version": "0.0.1",
```

Comment:
`0.0.1` isn't technically a valid semver version. The first version should be `0.1.0`.